node/node.go

```diff
@@ -938,7 +938,45 @@ func (n *Node) setTxGuardianData(guardian string, guardianSigHex string, tx *tra
 }
 
 // GetAccount will return account details for a given address
-func (n *Node) GetAccount(address string, options api.AccountQueryOptions, ctx context.Context) (api.AccountResponse, api.BlockInfo, error) {
+func (n *Node) GetAccount(address string, options api.AccountQueryOptions) (api.AccountResponse, api.BlockInfo, error) {
+	account, blockInfo, err := n.loadUserAccountHandlerByAddress(address, options)
```

Comment:
This first part of the code is duplicated from `GetAccount` -> `GetAccountWithKeys`.
I think we should extract it to a common base function

Reply:
fixed